js/client-chat.js

```diff
@@ -958,6 +958,13 @@
 					});
 					break;
 
+				case 'remove':
```

Comment:
I'm not sure we want to use as general a word as `remove`. We could repurpose `unlink` or something.